webpack_config/common.js

```diff
@@ -237,8 +237,8 @@ module.exports = {
       metaCsp: IS_DEV
         ? ''
         : `default-src 'none'; script-src 'self'; worker-src 'self' blob:; child-src 'self'; style-src 'self' 'unsafe-inline'; manifest-src 'self'; font-src 'self'; img-src 'self' data: https://mycryptoapi.com/api/v1/images/; connect-src *${
-        IS_ELECTRON ? ' eth-enclave:' : ''
-        }; frame-src 'self' https://connect.trezor.io;`
+            IS_ELECTRON ? ' eth-enclave:' : ''
+          }; frame-src 'self' https://connect.trezor.io https://landing.mycryptobuilds.com;`
```

Comment:
Do we need to add `mycrypto.com` then?

Reply:
Added `beta`. We should not need the feature when we reach `mycrypto.com`